### PR TITLE
Set replicas to default value

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <p.loffay@gmail.com>

Follow up for #798

### CODE REVIEW

File: apis/v1alpha1/opentelemetrycollector_webhook.go [hunk 1]
```diff
@@ -40,6 +40,8 @@ var _ webhook.Defaulter = &OpenTelemetryCollector{}
 
 // Default implements webhook.Defaulter so a webhook will be registered for the type.
 func (r *OpenTelemetryCollector) Default() {
+	opentelemetrycollectorlog.Info("default", "name", r.Name)
```
Comment (jpkrohling) [inline apis/v1alpha1/opentelemetrycollector_webhook.go:43]: Is this a leftover from a local test?

File: apis/v1alpha1/opentelemetrycollector_webhook_test.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestOELColDefaultingWebhook(t *testing.T) {
```
Comment (jpkrohling) [inline apis/v1alpha1/opentelemetrycollector_webhook_test.go:10]:
```suggestion
func TestOTELColDefaultingWebhook(t *testing.T) {
```

**Review by jpkrohling — APPROVED** [review]
LGTM, but I wonder if this is a new behavior brought in by one of our dependencies, or if we always had this bug and nobody ever complained, which seems unlikely, as I doubt all our users are setting the number of replicas in their CRs.

File: apis/v1alpha1/opentelemetrycollector_webhook.go (hunk 1 above)
Comment (pavolloffay) [reply apis/v1alpha1/opentelemetrycollector_webhook.go:43]: nope just moving the logging from the end of the function

Comment (jpkrohling) [reply apis/v1alpha1/opentelemetrycollector_webhook.go:43]: Would it make sense to change it somehow? Perhaps even remove it?

**Review by frzifus — APPROVED** [review]

File: apis/v1alpha1/opentelemetrycollector_webhook.go (hunk 1 above)
Comment (jpkrohling) [reply apis/v1alpha1/opentelemetrycollector_webhook.go:43]: Alright -- the level is set to "info", but it's not providing any real info. It sounds more like a debug from your description. I'll let it at your discretion though.

**Pull request merged**